plugins/techdocs/package.json

```diff
@@ -46,6 +46,7 @@
     "@backstage/plugin-search-react": "^0.2.0-next.1",
     "@backstage/plugin-techdocs-react": "^0.1.1-next.1",
     "@backstage/theme": "^0.2.15",
+    "@material-table/core": "^3.1.0",
```

Comment:
Has the yarn.lock file changed? If so, include it in the pull request so I can merge it 🙂

Reply:
added `yarn.lock`, it has changed a bit, but not on the package i added, thus I thought previously no need to include it. Sorry for that, cheers!

Reply:
there's an error in the previous pipeline - something related to kubernetes, so i attempted to retrigger the workflow in the hope that it has fixes itself overnight